platform_main_feature.strongarm.inc

```diff
@@ -493,6 +493,13 @@ function platform_main_feature_strongarm() {
   $strongarm->value = 'fgs_label_ss asc';
   $export['islandora_solr_collection_sort'] = $strongarm;
 
+  $strongarm = new stdClass();
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
+  $strongarm->name = 'islandora_solr_collection_sort';
```

Comment:
This is the same as https://github.com/discoverygarden/platform_main_feature/pull/224/files#diff-5a08a43b2c687be219ac25ecea6605d6L492?